src/resources/2020/Guzzlr.ts

```diff
@@ -1,5 +1,5 @@
 import { Item, Location, mallPrice, runChoice, use, visitUrl } from "kolmafia";
-import { maxBy } from "lodash";
+import maxBy from "lodash/maxBy";
```

Comment:
Is this general spring cleaning, or is this something that we'll specifically want to start doing once we switch to esbuild?

Reply:
We should be doing this now

Reply:
Otherwise we massively inflate the build size